README.md

```diff
@@ -25,6 +25,7 @@ This includes a number of key Agent-based Modelling (ABM) building blocks such a
 <!-- Remove this section once it is no longer in pre-release / alpha state -->
 
 FLAME GPU 2 is currently in an pre-release (alpha) state, and although we hope there will not be significant changes to the API prior to a stable release there may be breaking changes as we fix issues, adjust the API and improve performance.
+The use of *pure* Python agent functions (agent functions expressed as Python syntax which are transpiled to C++) is currently supported (see examples) but classed as an *experimental* feature.
```

Comment:
Still refers to `*pure* Python`, which has been removed from teh docs repo.

Reply:
ok fixed